code/modules/events/sayu_artifactdust.dm

```diff
@@ -0,0 +1,23 @@
+// ARTIFACT METEORS -- Collen
+
+/datum/round_event_control/meteor_wave/artifact
+	name = "Artifact Meteor"
+	typepath = /datum/round_event/meteor_wave/artifact
+	weight = 260 // This value needs some tweaking, not really sure what to put it at.
```

Comment:
this makes artifacts as common as *space dust*, happening every five minutes.
An actual event is usually around 5-10